pkg/inclusion/get_commit.go

```diff
@@ -4,9 +4,26 @@ import (
 	"errors"
 
 	"github.com/celestiaorg/celestia-app/pkg/da"
+	"github.com/celestiaorg/celestia-app/pkg/shares"
 	"github.com/tendermint/tendermint/crypto/merkle"
 )
 
+func GetMultiCommit(cacher *EDSSubTreeRootCacher, dah da.DataAvailabilityHeader, startIndexes, lengths []uint32) ([]byte, error) {
```

Comment:
[non-blocking] nit: Is there a rhyme or reason to the use of `commit` vs `commitment`?  I'd hate for us to run into some dumb bug that came from not finding all the right `commit` locations because we searched for `commitment`....  

I'd recommend a `chore` task to unify the naming to avoid any confusion in the future.

Reply:
The previous function was named commit, so this one was as well, but there is no reason we shouldn't be consistent across the board and not just per package

Reply:
opened #1202 :slightly_smiling_face: